src/sentry/dynamic_sampling/tasks/boost_low_volume_projects.py

```diff
@@ -112,31 +114,40 @@ def boost_low_volume_projects(context: TaskContext) -> None:
                 boost_low_volume_projects_of_org.delay(org_id, projects)
 
 
+@metrics.wraps("dynamic_sampling.partition_by_measure")
 def partition_by_measure(org_ids: list[OrganizationId]) -> Mapping[SamplingMeasure, list[int]]:
     """
     Partitions the orgs by the measure that will be used to adjust the sample
     rates. This is controlled through a feature flag on the organization,
     determined by its plan.
+
+    Only organizations with organization-mode sampling will be considered. In
+    project-mode sampling, the sample rate is set per project, so there is no
+    need to adjust the sample rates.
     """
 
+    original_orgs = Organization.objects.get_many_from_cache(org_ids)
+    modes = OrganizationOption.objects.get_value_bulk(original_orgs, "sentry:sampling_mode")
+
+    # Exclude orgs with project-mode sampling from the start. We know the
+    # default is DynamicSamplingMode.ORGANIZATION.
+    orgs = [org for org, mode in modes.items() if mode != DynamicSamplingMode.PROJECT]
+
+    if not options.get("dynamic-sampling.check_span_feature_flag"):
+        return {SamplingMeasure.TRANSACTIONS: [org.id for org in orgs]}
```

Comment:
I think that is okay, since we only need to rebalance orgs with manual `samplingMode`.

Reply:
> I think that is okay, since we only need to rebalance orgs with manual samplingMode.

With **automatic** sampling mode, but yes that's the reason. I wanted to keep the option to use spans as balancing input separate from the sampling mode.